projects/manager/src/app/datasets/loading-info/gdal-metadata-list/gdal-metadata-list.component.html

```diff
@@ -31,6 +31,13 @@
             <mat-label>Band Name</mat-label>
             <input matInput type="text" formControlName="bandName" />
         </mat-form-field>
+
+        <ng-container *ngIf="metaData as meta">
+            <geoengine-measurement
```

Comment:
I think it would be good to add an indication that the measurement is set in this section

Reply:
https://github.com/geo-engine/geoengine-ui/pull/788/commits/0dee455e72e64d482bd8c8118dc65ce5edafb606